modules/ossm-threescale-applying-external-service-entry-objects.adoc

```diff
@@ -50,23 +67,57 @@ spec:
   resolution: DNS
 ----
 +
-You can use the `oc apply` command with either of the following methods to apply the objects:
+.`DestinationRule` CR
+[source,terminal]
+----
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: destination-rule-threescale-saas-system
+spec:
+  host: multitenant.3scale.net
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
+      sni: multitenant.3scale.net
+----
++
+Use the following commands to apply external `ServiceEntry` and related `DestinationRule` CRs to your cluster:
++
+.3scale Hosted backend and system
```

Comment:
Check the preview, there are four bullets, but it looks like two of them are repeats?  Possibly a copy/paste error?

Reply:
These are different for system and backend. For example, the commands in step 1 vs step 3. The same applies to step 2 and step 4.

1. `$ oc apply -f <service-entry-threescale-saas-backend.yml>`
`$ oc apply -f <destination-rule-threescale-saas-backend.yml>`

3. $ oc apply -f <service-entry-threescale-saas-system.yml>
$ oc apply -f <destination-rule-threescale-saas-system.yml>

Reply:
There is no text that distinguishes between the two sets of bullets.  It's pretty much impossible to tell that one set of commands is intended for the  system and one for the backend. Is the text "3scale Hosted backend and system" supposed to be a heading (no space between the period and the text in the markup)?  Or is it supposed to be a procedure step (needs a space between the period and the text in the markup)?  It might be clearer to actually make this two separate steps, one for the system and one for the backend.

Reply:
Sorry about the lack of clarity. The text with no space between the period and the text creates a title or heading for that which is under it if I'm not mistaken. It is not a procedure step, which as we know require a space between the period and text.

In any case, I will make this clearer.

I have an issue with this PR. For the last commit, I am getting this message "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository." I do not know why it happened, but I think made changes based on the last comments from you and Alex, however none of them showed up in the commit.

I am going to have to close this PR and create a new one with the updates. My apologies for the inconvenience (if any) this causes. I think I've everything I need to progress with the PR now, including feedback from yourself and Alex. Hopefully nearly there. This has been delayed long enough.

Reply:
**Update:** My latest commit seems to have end up in this PR, although I started from scratch locally, so I guess I'll keep it open. Let's see if all the checks pass.